src/dotnet/APIView/APIViewWeb/Repositories/NotificationManager.cs

```diff
@@ -20,26 +21,82 @@ public class NotificationManager
+        private readonly CosmosUserProfileRepository _userProfileRepository;
 
         private const string ENDPOINT_SETTING = "Endpoint";
         private const string SENDGRID_KEY_SETTING = "SendGrid:Key";
         private const string FROM_ADDRESS = "apiview-noreply@microsoft.com";
         private const string REPLY_TO_HEADER = "In-Reply-To";
         private const string REFERENCES_HEADER = "References";
 
-        public NotificationManager(IConfiguration configuration, CosmosReviewRepository reviewRepository)
+        public NotificationManager(IConfiguration configuration, CosmosReviewRepository reviewRepository,
+            CosmosUserProfileRepository userProfileRepository)
         {
             _sendGridKey = configuration[SENDGRID_KEY_SETTING];
             _endpoint = configuration.GetValue<string>(ENDPOINT_SETTING);
             _reviewRepository = reviewRepository;
+            _userProfileRepository = userProfileRepository;
         }
 
         public async Task NotifySubscribersOnComment(ClaimsPrincipal user, CommentModel comment)
         {
             ReviewModel review = await _reviewRepository.GetReviewAsync(comment.ReviewId);
             await SendEmailsAsync(review, user, GetPlainTextContent(comment), GetHtmlContent(comment, review));
         }
+        
+        public async Task NotifyUserOnCommentTag(string username, CommentModel comment)
+        {
+            ReviewModel review = await _reviewRepository.GetReviewAsync(comment.ReviewId);
+            UserProfileModel user = await _userProfileRepository.tryGetUserProfileByNameAsync(username);
+            await SendUserEmailsAsync(review, user, GetCommentTagPlainTextContent(comment), GetCommentTagHtmlContent(comment, review));
+        }
+        
+        public async Task NotifyApproversOfReview(ClaimsPrincipal user, string reviewId, HashSet<String> reviewers)
+        {
+            UserProfileModel userProfile = await _userProfileRepository.tryGetUserProfileAsync(user);
+            ReviewModel review = await _reviewRepository.GetReviewAsync(reviewId);
+            foreach (string reviewer in reviewers)
+            {
+                UserProfileModel reviewerProfile = await _userProfileRepository.tryGetUserProfileByNameAsync(reviewer);
+                await SendUserEmailsAsync(review, reviewerProfile, 
+                    GetApproverReviewContentHeading(userProfile,false), 
+                    GetApproverReviewHtmlContent(userProfile, review));
+            }
+        }
+        
+        private string GetApproverReviewHtmlContent(UserProfileModel user, ReviewModel review)
+        {
+            var reviewName = review.Name;
+            var reviewLink = new Uri($"{_endpoint}/Assemblies/Review/{review.ReviewId}");
+            var poster = user.UserName;
+            var userLink = new Uri($"{_endpoint}/Assemblies/Profile/{poster}");
+            var requestsLink = new Uri($"{_endpoint}/Assemblies/RequestedReviews/RequestedReviews");
```

Comment:
Is this link correct? /Assemblies/RequestedReviews/**RequestedReviews**

Reply:
Corrected this also